extensions/blocks/contact-form/edit.js

```diff
@@ -47,16 +58,15 @@ const ALLOWED_BLOCKS = [
 	'core/video',
 ];
 
-class JetpackContactForm extends Component {
+class JetpackContactFormEdit extends Component {
```

Comment:
Ah, I sure hoped to see this converted into a functional component as well! 🙂

Reply:
I've converted most of the individual fields and I almost started on this yesterday but had second thoughts based on getting this PR merged first. I could do a follow up PR, or do you think it's worth doing here?

Reply:
You inspired me, so I rewrote it as a functional component. 😀

Reply:
I should have kept my mouth shut! Now I have to review that as well! 😄